### ISSUE TITLE
Error while restarting training. 

### ISSUE THREAD

**Issue by evilc3** [NVIDIA/NeMo#4911]
Getting the following error : Early stopping conditioned on metric `val_loss` which is not available. Pass in or modify your `EarlyStopping` callback to use any of the following: ``

using nemo-toolkit['asr'] == 1.10.0

pytorch-lightning       1.7.3
torch                            1.12.1
torchmetrics               0.9.3

looks like this is a Pytorch-lightning help any work around this?

    trainer = ptl.Trainer(
                          val_check_interval= FLAGS.val_check_interval,
                          devices=-1, 
                          max_epochs=EPOCHS, 
                          accumulate_grad_batches=FLAGS.accumulate_grad_batches,
                          enable_checkpointing=FLAGS.enable_checkpointing,
                          logger=FLAGS.logger,
                          log_every_n_steps=FLAGS.log_every_n_steps, ##value should be high to save memory.
                         callbacks = [early_stop],
                          sync_batchnorm = FLAGS.sync_batchnorm,
                          gradient_clip_val=FLAGS.gradient_clip_val,
                          num_sanity_val_steps=0,
                          )

@titu1994

**Comment by evilc3** [comment]
one way I found around this  was by removing the callback

**Comment by evilc3** [comment]
Hi @titu1994  I have successfully resumed training, the only change made is have added a new training dataset.
